TACC/TACC-Docs
# fix(docs): RT-88742 guassian_user_agreement PDF broken link

A link on Guassian document is lowercase, but filename was uppercase.

## Overview

…

## Related

- [RT-88742](https://consult.tacc.utexas.edu/Ticket/Display.html?id=88742)

## Changes

- rename guassian user agreement pdf

## Testing

1. Open http://0.0.0.0:8000/software/gaussian/.
2. Click "User Agreement" link.
3. Verify document loads.

## UI

https://user-images.githubusercontent.com/62723358/234105766-97012464-f437-47c7-980e-ece9fa81eb13.mov
